### PR TITLE
Feature 3: Infographics

### PR DESCRIPTION
**Pull request by JustOneMoreLine**
Infographics berupa:
 a. Banyak rumah sakit yang sudah memiliki ER
 b. Banyak rumah sakit untuk tiap rating "patient experience"
 c. Banyak rumah sakit untuk tiap rating "overall rating"

### CODE REVIEW

File: hospitals/views.py [hunk 1]
```diff
@@ -21,6 +25,78 @@ def index(request):
 	
 	return render(request, 'index.html', index_context)
 
+prefixes = '''
```
Comment (gpersable) [inline hospitals/views.py:28]: Please refer to `utils.py` for querying. Thanks.

**Review by gpersable — COMMENTED** [review]
Please refer to `utils.py` for querying. Thanks.

File: hospitals/utils.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from pymantic import sparql
 
-server = sparql.SPARQLServer('http://localhost:9999/blazegraph/namespace/kb/sparql')
+server = sparql.SPARQLServer('http://169.254.0.72:9999/blazegraph/namespace/kb/sparql')
```
Comment (dnswd) [inline hospitals/utils.py]: Maybe we can use environment variable instead?

File: hospitals/views.py [hunk 3]
```diff
@@ -21,6 +22,37 @@ def index(request):
...
+	PREFIX : <http://example.org/>
+	PREFIX schema: <http://schema.org/>
+	PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+	PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+'''
+def infographics(request):
+
+	# Make query calls from utils.py, and extract them
```
Comment (dnswd) [inline hospitals/views.py:28]: I think we should agree on what indentation to use lol, but it would work for now

**Review by dnswd — APPROVED** [review]
LGTM

File: hospitals/utils.py (hunk 2 above)
Comment (JustOneMoreLine) [reply hospitals/utils.py]: we should :v

Comment (gpersable) [reply hospitals/utils.py]: Change to localhost :v

**Pull request merged**